build/util.sh

```diff
@@ -23,8 +23,15 @@ function kube::release::semantic_version() {
   # Client Version: version.Info{Major:"1", Minor:"1+", GitVersion:"v1.1.0-alpha.0.2328+3c0a05de4a38e3", GitCommit:"3c0a05de4a38e355d147dbfb4d85bad6d2d73bb9", GitTreeState:"clean"}
   # and spits back the GitVersion piece in a way that is somewhat
   # resilient to the other fields changing (we hope)
-  ${KUBE_ROOT}/cluster/kubectl.sh version --client | sed "s/, */\\
-/g" | egrep "^GitVersion:" | cut -f2 -d: | cut -f2 -d\"
+  local GIT_VERSION=$(${KUBE_ROOT}/cluster/kubectl.sh version --client | sed "s/, */\\
+/g" | egrep "^GitVersion:" | cut -f2 -d: | cut -f2 -d\")
+
+  # In case kubectl not available
+  if [[ -z ${GIT_VERSION} ]]; then
```

Comment:
@shashidharatd I have two concerns about this change.

1. Where are we calling this function before building `kubectl`? Shouldn't we fix the caller to have a `kubectl` built instead fixing it here?
2. Who supplies `$KUBE_GIT_VERSION`? In other words, what guarantees that it is set and it is set to the right values?